starter_code/index.html

```diff
@@ -14,7 +15,7 @@
   <section class="container">
     <h1>Ironhack Characters API</h1>
     <div class="characters-container">
-      <div class="character-info">
+      <div class="character-info" id='character-info-template'>
```

Comment:
You don't really need that `id`. I bet you can figure a selector for that element.